frontend/simple/components/MembersCircle.vue

```diff
@@ -1,90 +1,197 @@
-
-  > * {
-    display: block;
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    width: 6em;
-    height: 6em;
-    margin: -3em; // half previous value
+}
+
+@mixin centerXY {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.gi-memberCircle-container,
+.gi-svg,
+.gi-circle-list {
+  position: relative;
+  width: $mainCircle-size;
+  height: $mainCircle-size;
+}
+
+.gi-svg {
+  position: absolute;
+
+  &-circle {
+    stroke: $primary;
+    stroke-width: $border-width;
+    stroke-dasharray: $gi-spacer*0.75 $gi-spacer;
+    stroke-linecap: round;
+    fill: transparent;
+  }
+}
+
+.gi-item {
+  width: $itemCircle-size;
+  height: $itemCircle-size;
+  @include centerXY;
+
+  &-box {
+    position: relative;
+    width: 100%;
+    height: 100%;
+
+    &::before {
+      position: absolute;
+      content: '';
+      box-shadow: 0 0 0 $border-width $primary;
+      opacity: 0;
+      transition: opacity 150ms;
+      @include itemRound;
+    }
+
+    &.is-active::before,
+    &:hover::before {
+      opacity: 1;
+    }
+  }
+
+  &-img {
+    border: $border-width solid $body-background-color;
+    @include itemRound;
   }
 
-  img {
-    display: block;
+  &-others {
     width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
     border-radius: 50%;
-    box-shadow: 0 0 0 5px #64caf1;
+    background: $white-ter;
+    color: $primary;
   }
+}
 
-  // a {
-  //   display: block;
-  //   border-radius: 50%;
-  //   box-shadow: 0 0 0 5px tomato;
-  // }
+.gi-center {
+  max-width: $mainCircle-size - $itemCircle-size - $gi-spacer-lg;
+  @include centerXY;
 }
 
+.gi-memberInfo {
+  &-desc {
+    margin: $gi-spacer-xs 0 $gi-spacer-sm;
+  }
+
+  &-tag {
+    background: rgba($primary, 0.3);
+  }
+}
 </style>
 <script>
-import {mapGetters} from 'vuex'
 export default {
   name: 'MembersCircle',
-  mounted () {
+  props: {
+    members: Array
+  },
+  data () {
+    return {
+      memberActive: false,
+      maxMembers: 6
+    }
   },
   methods: {
-
+    toggleMemberActive (member, toActive = !this.memberActive) {
+      this.memberActive = toActive ? member : false
+    }
   },
   computed: {
-    members () {
-      var members = this.$store.getters.profilesForGroup()
-      const usernames = Object.keys(members)
-      const thetaIncr = Math.PI * 2.0 / usernames.length
-      usernames.map(function (username, idx) {
-        var mt = 10 * Math.sin(thetaIncr * idx) // r * sin(theta)
-        var ml = 10 * Math.cos(thetaIncr * idx) // r * sin(theta)
-        members[username] = {
-          attrs: members[username].globalProfile,
+    membersWithStyle () {
+      const { members } = this
+      const thetaIncr = Math.PI * 2.0 / members.length
+      let mt = 0
+      let ml = 0
+
+      members.forEach((member, i) => {
+        mt = 9.5 * Math.sin(thetaIncr * i) // r * sin(theta)
+        ml = 9.5 * Math.cos(thetaIncr * i) // r * sin(theta)
```

Comment:
Maybe these "r"'s (radiuses) should be based on something else?

How should this view react when resized to mobile-size for example? Currently it isn't responsive but fixed to `9.5`. Is that something you'd like to address in this PR or in a future PR? If future PR, please open an issue for it so we don't forget.

Reply:
I'll look into this in this same PR

Reply:
I've update it to be responsive, actually this value is the `radiusCircle`, so it should be `10` after tablet. Until it is smaller, `8`.

Once #408 is merged, `isTablet` (and other sizes) can be created at `utils/responsive.js`.